src/counterResults.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const { getValueInput } = require('./getValueInput.js');
+
+function counterResults(input, random) {
+  const result = {
+    cows: 0,
+    buls: 0,
+  };
+
+  const validatedNumber = getValueInput(input);
+
+  for (let i = 0; i < validatedNumber.length; i++) {
+    const isEqualNumber = validatedNumber[i] === random[i];
+    const isIncludeNumber = random.includes(validatedNumber[i]);
+
+    if (isEqualNumber) {
+      result.cows++;
+    } else if (isIncludeNumber) {
```

Comment:
can you avoid `if / else if` construction?

Reply:
:(

Reply:
?